docker/Makefile

```diff
@@ -15,15 +15,18 @@
 
 DOCKER_BUILD_OPTS=
 DOCKER_COMPOSE_COMMAND=docker-compose
-DOCKER_COMPOSE_FILES=
-DOCKER_COMPOSE_OPTS=
+DOCKER_COMPOSE_FILES_STL=
+DOCKER_COMPOSE_FILES_CCF=
+DOCKER_COMPOSE_OPTS_STL=
+DOCKER_COMPOSE_OPTS_CCF=
 
 # optionally allow local overriding defaults
 -include make.loc
 
-DOCKER_COMPOSE_FILES += sawtooth-pdo.yaml sawtooth-pdo.local-code.yaml
+DOCKER_COMPOSE_FILES_STL += sawtooth-pdo.yaml sawtooth-pdo.local-code.yaml
+DOCKER_COMPOSE_FILES_CCF += ccf-pdo.yaml ccf-pdo.local-code.yaml
 ifeq ($(SGX_MODE),HW)
-   DOCKER_COMPOSE_FILES += sawtooth-pdo.sgx.yaml
+   DOCKER_COMPOSE_FILES_STL += sawtooth-pdo.sgx.yaml
```

Comment:
afaik, ccf does not support HW mode yet?  Should we maybe fail in that case? or at least send a warning?  Right now we just seem to silently ignore the SGX_MODE var and always run in SIM mode for ccf ...

Reply:
you are right, ccf/pdo combo tested now only with SIM mode for both pdo & ccf. As for using HW mode in pdo, the pdo-tp -ccf needs to be augmented with support for proof data check (so if you set PDO SGX_MODE to HW), pdo-tp-ccf will throw an error saying that proof data check is currently unsupported. pdo-tp-ccf should work in HW mode with no changes (I checked once in ACC), but perhaps I would like to retest & document before making the claim.

Reply:
> you are right, ccf/pdo combo tested now only with SIM mode for both pdo & ccf. As for using HW mode in pdo, the pdo-tp -ccf needs to be augmented with support for proof data check (so if you set PDO SGX_MODE to HW), pdo-tp-ccf will throw an error saying that proof data check is currently unsupported. pdo-tp-ccf should work in HW mode with no changes (I checked once in ACC), but perhaps I would like to retest & document before making the claim.

I tested `make -C docker test` in HW mode and it runs through. Makes sense as for ccf you essentially ignore SGX_MODE for both build (of ccf) and runs (for both CCF and rest of PDO, i.e., EHS/SS/PS) with SGX_MODE=SIM (through the default which is not overriden in ...sgx.yaml to point to the sgx rather than sim version of the container).